### PR TITLE
[IMP] purchase_partner_incoterm: use commercial_partner_id

### PR DESCRIPTION
**Pull request by hparfr**
Incoterms should be on the supplier, not on the contact.

### CODE REVIEW

**Comment by TDu** [comment]
Make sense, but we may need a migration script for it.

File: purchase_partner_incoterm/models/res_partner.py [hunk 1]
```diff
@@ -16,3 +16,10 @@ class ResPartner(models.Model):
         ondelete="restrict",
         string="Default Purchase Incoterm Address",
     )
+
+    @api.model
+    def _commercial_fields(self):
+        return super(ResPartner, self)._commercial_fields() + [
```
Comment (bealdav) [inline purchase_partner_incoterm/models/res_partner.py]:
```suggestion
        return super()._commercial_fields() + [
```

**Review by HviorForgeFlow — APPROVED** [review]

**Review by bealdav — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@hparfr I'm not sure if applying major merge will overwrite your changes on version's manifest. Could you undo undo that version change and I apply major merge?

**Comment by hparfr** [comment]
> @hparfr I'm not sure if applying major merge will overwrite your changes on version's manifest. Could you undo undo that version change and I apply major merge?

The hardcoded version change in the manifest is to ensure the naming of the migration script.
I think you can use `ocabot merge nobump`

**Pull request merged**